common/sketch/src/main/java/org/apache/spark/util/sketch/BloomFilter.java

```diff
@@ -233,4 +233,5 @@ public static BloomFilter create(long expectedNumItems, long numBits) {
 
     return new BloomFilterImpl(optimalNumOfHashFunctions(expectedNumItems, numBits), numBits);
   }
+
```

Comment:
Go ahead and back out this spurious change; there's another newline at the end of the next file